src/main/java/mate.academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
Ensure that the `Injector` class is correctly configured to map the `ProductService` interface to its implementation. This is crucial for the `getInstance` method to work correctly and avoid runtime exceptions. Refer to the previous review for guidance on using a `Map` for interface-to-implementation mapping.